packages/node/src/login/oidc/refresh/TokenRefresher.ts

```diff
@@ -80,9 +82,16 @@ export default class TokenRefresher implements ITokenRefresher {
       { sessionId },
       oidcContext.issuerConfig
     );
+    if (clientInfo.idTokenSignedResponseAlg === undefined) {
```

Comment:
Likewise here as the other comment about comparing with `undefined`.

Reply:
a90e36b19f5ac153d425356eb01a746b5f52f1a5